frontend/src/Editor/LeftSidebar/SidebarInspector.jsx

```diff
@@ -16,7 +17,8 @@ const staticDataSources = [
 
 export const LeftSidebarInspector = ({
   darkMode,
-  currentState,
+  selectedSidebarItem,
+  setSelectedSidebarItem,
```

Comment:
If it is not used anywhere we can remove it

Reply:
Done